ArchiSteamFarm/Actions.cs

```diff
@@ -244,7 +244,7 @@ public sealed class Actions : IAsyncDisposable {
 		}
 
 		[PublicAPI]
-		public async Task<(bool Success, string Message)> SendInventory(uint appID = Steam.Asset.SteamAppID, ulong contextID = Steam.Asset.SteamCommunityContextID, ulong targetSteamID = 0, string? tradeToken = null, Func<Steam.Asset, bool>? filterFunction = null) {
+		public async Task<(bool Success, string Message)> SendInventory(uint appID = Steam.Asset.SteamAppID, ulong contextID = Steam.Asset.SteamCommunityContextID, ulong targetSteamID = 0, string? tradeToken = null, Func<Steam.Asset, bool>? filterFunction = null, IReadOnlyCollection<Steam.Asset>? inventory = null) {
```

Comment:
This function does different thing now than what it was designed to do. Split it into two `[PublicAPI]` functions, one taking filter and second one taking inventory, and make filter one call inventory one with common logic.

Reply:
Forgot to push that previously. Done